code/script.js

```diff
@@ -50,49 +55,49 @@ const CHARACTERS = [
     img: 'images/james.svg',
     hair: 'brown',
     eyes: 'green',
-    accessories: ['glasses'],
-    other: []
+    accessories: ['sunglasses'],
+    other: ['beard']
   },
   {
     name: 'Jana',
     img: 'images/jana.svg',
     hair: 'black',
     eyes: 'hidden',
-    accessories: ['glasses'],
-    other: []
+    accessories: ['sunglasses'],
+    other: ['jewelleries']
   },
   {
     name: 'Jane',
     img: 'images/jane.svg',
-    hair: 'yellow',
+    hair: 'blonde',
     eyes: 'hidden',
-    accessories: ['glasses'],
+    accessories: ['sunglasses'],
```

Comment:
Nice that you split up glasses to be sunglasses and just regular glasses, did that too :)